addons/website_slides/controllers/main.py

```diff
@@ -321,6 +325,18 @@ def create_slide(self, *args, **post):
             return {'error': _('Internal server error, please try again later or contact administrator.\nHere is the error message: %s') % e.message}
         return {'url': "/slides/slide/%s" % (slide_id.id)}
 
+    @http.route("/slides/max_size", type="json", auth="public", website=True)
```

Comment:
I don't like the idea to add a controller just to get such minor information :-/ can't we get it somewhere else?

Reply:
maybe you could just add this max-size during rendering of views where snippet is used.
Btw, a controller can be intercepted, so it stay a pure client side check, so not risk imo

Reply:
I cannot push anymore here. Can you add the fixes, or do I open a new PR?